Add-ons/UmbracoForms/Developer/Extending/Adding-a-Type.md

```diff
@@ -181,9 +181,9 @@ public class TextareaWithCount : Umbraco.Forms.Core.Providers.FieldTypes.Textare
         this.Name = "Long Answer with Limit";
     }
 
-    public override IEnumerable<string> ValidateField(Form form, Field field, IEnumerable<object> postedValues, HttpContextBase context, IFormStorage formStorage)
+    public override IEnumerable<string> ValidateField(Form form, Field field, IEnumerable<object> postedValues, HttpContext context, IPlaceholderParsingService placeholderParsingService, List<string> errors)
```

Comment:
Thanks, this got omitted from being updated from the Forms 8 version by the looks of it, so good to have it correct now.